jquery.leadservice.js

```diff
@@ -220,19 +221,14 @@ define(['jquery', 'jquery.cookie'], function($) {
           type: 'POST',
           data: $(this).serialize(),
           success: function(response) {
-            status = 'success';
-            opts.lead_saved();
+            var data = opts.lead_saved();
+            $('body').trigger('lead_submission', (data || {}));
```

Comment:
not a big deal, it think this looks cleaner `var data = opts.lead_saved() || {};`